packages/core/src/hub.ts

```diff
@@ -262,27 +263,26 @@ export class Hub implements HubInterface {
 
   /**
    * @inheritDoc
+   *
+   * @deprecated Use top level `captureException` instead.
```

Comment:
```suggestion
   * @deprecated Use `Sentry.captureException()` instead.
```

maybe a bit clearer?